_data-prepper/pipelines/configuration/processors/add-entries.md

```diff
@@ -28,31 +28,26 @@ To get started, create the following `pipeline.yaml` file:
 ```yaml
 pipeline:
   source:
-    file:
-      path: "/full/path/to/logs_json.log"
-      record_type: "event"
-      format: "json"
+    ...
+  ....  
   processor:
     - add_entries:
         entries:
         - key: "newMessage"
           value: 3
           overwrite_if_key_exists: true
   sink:
-    - stdout:
 ```
 {% include copy.html %}
 
 
-Next, create a log file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` file with that filepath. For more information, see [Configuring Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/getting-started/#2-configuring-data-prepper).
-
-For example, before you run the `add_entries` processor, if the `logs_json.log` file contains the following event record:
+For example, before you run the `add_entries` processor, when your source contains the following event record:
```

Comment:
This sentence needs rewriting, the meaning is not clear.

Reply:
Removed the "before" clause